build.gradle

```diff
@@ -117,17 +123,20 @@ subprojects { subproject ->
 // BEGIN INDIVIDUAL PROJECTS
 project(':kcbq-connector') {
     apply plugin: 'jacoco'
-
     jar {
         manifest {
             attributes  'Implementation-Title': 'Kafka Connect BigQuery Connector',
-                        'Implementation-Version': version
+                    'Implementation-Version': version
 
         }
     }
 
     repositories {
+        mavenLocal()
         mavenCentral()
+        maven {
+            url 'http://packages.confluent.io/maven'
+        }
```

Comment:
there are a lot of things in the file that seem unrelated to the PR. I know some of those are to get things working locally, but maybe we can trim things as much as possible to get it to pass CI, or maybe someone from the connect team can help figure this out (cc @C0urante)

Reply:
@xvrl feel free to ping Mikkin to see if you can get some resources from the Connect team, I don't have bandwidth for this right now but there's probably someone on the team who can help.